### PR TITLE
Fixes OptionsGeneration Test failing

### PR DESCRIPTION
**Pull request by mohsin**
As [mentioned here](https://github.com/octobercms/october/commit/9dca130faee47d598cc6f4e6febe337a652e1f4d#commitcomment-43513975), this fixes the test failure.

### CODE REVIEW

File: tests/unit/backend/widgets/FormTest.php [hunk 1]
```diff
@@ -160,7 +165,7 @@ public function testOptionsGeneration()
             'fields' => [
                 'static_method_options' => [
                     'type' => 'dropdown',
-                    'options' => '\FormHelper::staticMethodOptions',
+                    'options' => 'staticMethodOptions',
```
Comment (LukeTowers) [inline tests/unit/backend/widgets/FormTest.php]:
```suggestion
                    'options' => 'FormHelper::staticMethodOptions',
```

**Comment by LukeTowers** [comment]
@SaifurRahmanMohsin what steps are you taking to cause the test to fail? It's working fine for me locally.

**Comment by mohsin** [comment]
@LukeTowers Just these:
```
git clone http://github.com/octobercms/october
composer install
./vendor/bin/phpunit tests/unit/backend/
```

It still fails after changing `'\FormHelper::staticMethodOptions',` to `'FormHelper::staticMethodOptions',`. Do I need some additional setup for the tests and am doing something incorrectly? (running PHP 7.3.23).

**Comment by LukeTowers** [comment]
Works just fine for me @SaifurRahmanMohsin, running 7.4.3 & PHPUnit 8.5.8

**Comment by mohsin** [comment]
@LukeTowers I switched to PHP 7.4.11 and ran it on PHPUnit 8.5.8 and still get this issue. I am confused as to why this is happening to me alone.
Put a demo here >> https://youtu.be/0LFYI1lOSXg

File: tests/unit/backend/widgets/FormTest.php [hunk 2]
```diff
@@ -20,6 +20,11 @@ public function getDropdownOptions()
     {
         return ['dropdown', 'options'];
     }
+
+    public function staticMethodOptions()
+    {
+        return ['static', 'method'];
+    }
```
Comment (LukeTowers) [inline tests/unit/backend/widgets/FormTest.php:27]: @bennothommo can you revert this section? It shouldn't be required.

**Pull request merged**